website/docs/chatr.mdx

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_position: 6
+slug: /chatr
+title: Chatr
+tags: [JavaScript, Java, Python, C#, iOS, Kotlin]
+description: Create a doc page with rich content.
+hide_title: true
+---

```

Comment:
no content?

Reply:
I added content to this page. Same for the other pages of demo.